changelogs/fragments/2249-linode_v4-support-private_ip-option.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - linode_v4 - add support for ``private_ip`` option.
```

Comment:
```suggestion
  - linode_v4 - add ``private_ip`` option (https://github.com/ansible-collections/community.general/pull/2249).
```

Reply:
Done, thanks for your patience, I missed that part.

Reply:
No problem at all!